lib/tasks/csv_load.rake

```diff
@@ -15,6 +15,7 @@ namespace :csv_load do
     file_path = "db/data/merchants.csv"
     Merchant.destroy_all
     CSV.foreach(file_path, headers: true) do |row|
+      row[:status] = 0
```

Comment:
Nice! This is what we were lacking in adding a default to ours. This will help us set a default value for status